### ISSUE TITLE
Lazy loading documentation

### ISSUE THREAD

**Issue by divega** [aspnet/EntityFramework.Docs#612]
Let's get a draft of the docs for this feature in the 2.1.0-preview1 time frame. We either add the contents into the existing section in [related data](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/querying/related-data.md#lazy-loading) or create a new page if it is too long.

**Comment by bricelam** [comment]
Update the EF6 comparison table too

**Comment by divega** [comment]
Good point. I think the doc part is probably done. Added the help wanted just to make sure we are covering what we want and and close it.
